### PR TITLE
Update fa.rs

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/lang/fa.rs [hunk 1]
```diff
@@ -124,14 +124,14 @@ pub static ref T: std::collections::HashMap<&'static str, &'static str> =
         ("ScrollAuto", "پیمایش/اسکرول خودکار"),
         ("Good image quality", "کیفیت خوب تصویر"),
         ("Balanced", "متعادل"),
-        ("Optimize reaction time", "بهینه سازی زمان واکنش"),
+        ("Optimize reaction time", "بهینه سازی : زمان واکنش"),
```
Comment (rustdesk) [inline src/lang/fa.rs:127]: `:`？

File: src/lang/fa.rs [hunk 2]
```diff
@@ -483,7 +483,7 @@ pub static ref T: std::collections::HashMap<&'static str, &'static str> =
         ("System Sound", "صدای سیستم"),
         ("Default", "پیش فرض"),
         ("New RDP", "ریموت جدید"),
-        ("Fingerprint", "اثر انگشت"),
+        ("Fingerprint", "\n اثر انگشت"),
```
Comment (rustdesk) [inline src/lang/fa.rs:486]: `\n`?

Comment (ghost) [reply src/lang/fa.rs:486]: image
need `\n ` to show correct.

File: src/lang/fa.rs (hunk 1 above)
Comment (ghost) [reply src/lang/fa.rs:127]: use to better understanding

**Pull request merged**